build.gradle

```diff
@@ -460,6 +460,18 @@ task release() {
     }
 }
 
+// Publish to ~/.m2
+task releaseLocal() {
+    // This is necessary to upload the executable jar to bintray as well.
+    // It makes to run _bintrayRecordingCopy task after cli task.
+    release.mustRunAfter cli
+    dependsOn subprojects.publishToMavenLocal
+    doLast {
+        println "Released ${project.version}."
```

Comment:
How about adding `to local repository` or something?

Reply:
Good point. I will update it.